firmware/Core/Src/littlefs/littlefs_helper.c

```diff
@@ -14,8 +14,9 @@
 // Variables to track LittleFS on Flash Memory Module
 uint8_t LFS_is_lfs_mounted = 0;
 
-#define FLASH_CHIP_PAGE_SIZE_BYTES 512
-#define FLASH_CHIP_BLOCK_SIZE_BYTES 262144
+// NAND Flash Memory Datasheet https://www.farnell.com/datasheets/3151163.pdf
+#define FLASH_CHIP_PAGE_SIZE_BYTES 2048
```

Comment:
same comment as above, might be good to note that its 2176 but last 128 are pad bytes